app/assets/stylesheets/frontend/helpers/_organisations.scss

```diff
@@ -936,6 +936,10 @@ img.organisation-logo-custom {
   @extend .organisation-logo-identity-medium;
   border-left: none;
 }
+.organisation-logo-stacked-custom-medium {
```

Comment:
The styles in this block are identical to the ones above? Should they be combined?